docker/test/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# FIXME: once image is published inherit from official image
+# FROM cantino/huginn
+FROM singleprocess_huginn_web
```

Comment:
if `scripts/setup_env` were to be included in the official image, the test image could derive from the standard image